scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -92,10 +92,10 @@ export class Processor {
         this._handler = handler
     }
 
-    public process(input: TProtocol, output: TProtocol) {
+    public process(input: TProtocol, output: TProtocol, context?: any) {
```

Comment:
Instead of using any how about a type parameter? It should implicitly cast to any if unset.